app/assets/javascripts/lib/models/input_element.coffee

```diff
@@ -7,6 +7,7 @@ class @InputElement extends Backbone.Model
     @bind('change:user_value', @markDirty)
     @bind('change:user_value', @logUpdate)
     @bind('change:user_value', @update_collection)
+    @bind("change:user_value", App.analytics.changeInput)
```

Comment:
@antw I'm not sure how to add this event to the `InputElementList` model. Also, why are the other event listeners on the `InputElement` and not on the List?

Reply:
> Also, why are the other event listeners on the InputElement and not on the List?

Good question, I don't have an answer for that. 😄

Perhaps just leave it as part of InputElement then? At least it's consistent with the other events.